### PR TITLE
Move FrameTreeId type to gfx_traits #9221

### LINKED ISSUE #9221: Move FrameTreeId type to gfx_traits
It's used in compositing, gfx_traits, and gfx, but it's defined in msg. It should be possible to move it to gfx_traits instead.

Code: `components/msg/compositor_msg.rs`, `components/gfx_traits/lib.rs`

### PR DESCRIPTION
**Pull request by mskrzypkows**
Fixes #9221.

[Review on Reviewable](https://reviewable.io/reviews/servo/servo/9289)

### CODE REVIEW

File: components/gfx/paint_thread.rs [hunk 1]
```diff
@@ -16,11 +16,10 @@ use euclid::size::Size2D;
 use font_cache_thread::FontCacheThread;
 use font_context::FontContext;
 use gfx_traits::color;
-use gfx_traits::{Epoch, LayerId, LayerKind, LayerProperties, PaintListener, PaintMsg as ConstellationMsg, ScrollPolicy};
+use gfx_traits::{Epoch, FrameTreeId, LayerId, LayerKind, LayerProperties, PaintListener, PaintMsg as ConstellationMsg, ScrollPolicy};
```
Comment (KiChjang) [inline components/gfx/paint_thread.rs]: This line here fails tidy. Lines cannot be over 120 characters. Use `./mach test-tidy` to check whether your code conforms to tidy rules.

Comment (mskrzypkows) [inline components/gfx/paint_thread.rs]: It's strange, when I want to divide line:
'''
use gfx_traits::{color, Epoch, FrameTreeId};
use gfx_traits::{LayerId, LayerKind, LayerProperties, PaintListener, PaintMsg as ConstellationMsg, ScrollPolicy};
'''
I have an error:
''''
./components/gfx/paint_thread.rs:19: use statement is not in alphabetical order
        expected: gfx_traits::{color, Epoch, FrameTreeId}
        found: gfx_traits::{LayerId, LayerKind, LayerProperties, PaintListener, PaintMsg as ConstellationMsg, ScrollPolicy}
'''
How to divide it?

Comment (KiChjang) [inline components/gfx/paint_thread.rs]: Switch the two line positions.

Comment (nox) [inline components/gfx/paint_thread.rs]: First uppercase, then lowercase.

```rust
use gfx_traits::{Epoch, FrameTreeId, LayerId, LayerKind, LayerProperties, PaintListener};
use gfx_traits::{PaintMsg as ConstellationMsg, ScrollPolicy, color};
```

Comment (mskrzypkows) [inline components/gfx/paint_thread.rs]: OK, thx! I thought that it's not case sensitive.

**Pull request merged**